test/sframe/sframe_bench_aggregate.cpp

```diff
@@ -0,0 +1,159 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+#include <atomic>
+#include <thread>
+#include <core/storage/sframe_data/groupby_aggregate.hpp>
+#include <core/storage/sframe_data/groupby_aggregate_operators.hpp>
+#include <core/storage/sframe_data/sframe.hpp>
+#include <core/storage/sframe_data/sframe_constants.hpp>
+#include <iostream>
+
+namespace turi {
+
+using row_gen_func_t = std::function<std::vector<flexible_type>(size_t)>;
+
+/* generate sframe */
+/* I need to make this a fixture */
+static sframe bench_test_sframe_generator(
```

Comment:
Ah, thought you knew about this function: https://github.com/apple/turicreate/blob/master/src/core/storage/sframe_data/testing_utils.hpp#L72.  Let's use that here.

Reply:
ha! glad to know this one!